ex/high_throughput_inference/mock_app.py

```diff
@@ -183,19 +155,32 @@ def name(self):
 
     parser = argparse.ArgumentParser("Mock application")
     parser.add_argument("--device", default="cpu")
+    parser.add_argument("--log_max_batchsize", default=8, type=int)
     args = parser.parse_args()
 
     resnet = ResNetWrapper("resnet50", f"resnet50.{args.device.upper()}.pt")
 
     client = ProtoClient(timing_on=True)
     client.set_model(resnet.name, resnet.model)
 
-    total_iterations = 100
+    pt_model = torch.jit.load(io.BytesIO(initial_bytes=(resnet.model))).to("cuda:0")
```

Comment:
we have a device parameter but automatically use `cuda:0` device. do we want to make use of `args.device` and default to `cuda:0` instead of `cpu`?

Reply:
Yep